### ISSUE TITLE
MO Path Validation Not Always Accurate

### ISSUE THREAD

**Issue by bcrotty** [matortheeternal/merge-plugins#141]
819865d added validation to the Integrations tab, and it looks for ModOrganizer.ini in the selected MO directory. My ```ModOrganizer.ini``` is in ```C:\Users\<username>\AppData\Local\ModOrganizer\SkyrimSE```, and my ```ModOrganizer.exe``` is in ```G:\Games\Mod Organizer 2```. As I understand it, I need to select the location of the executable file, but validation is preventing me from doing that.

Also, can you confirm that the merge destination directory should point to the overwrite folder for MO, or does it matter?

Thanks!
Bobby

**Comment by matortheeternal** [comment]
MO2 has two ways it can be installed, portable or instance.  When installed with instance mode there isn't a way for Merge Plugins to know what the active profile is, where this was previously determined by looking at `ModOrganizer.ini`.  MO2 must be installed in portable mode to work with Merge Plugins.  A potential future feature would allow a user to specify the active MO profile in Merge Plugins manually when working with MO2 instances.

**Comment by Emilgardis** [comment]
Are there any plans on implementing this feature?

**Comment by matortheeternal** [comment]
@Emilgardis no, Merge Plugins is end of life.  I will no longer be updating or supporting it.

**Comment by ogrotten** [comment]
Is there a workaround way to have instance mode MO2 to work with Merge Plugins?

**Comment by theCalcaholic** [comment]
As a workaround I have created a symbolic link from Path\To\Mod Organizer 2\ModOrganizer.ini to %USERPROFILE%\AppData\Local\ModOrganizer\ModOrganizer.ini. 
It works on first glance, but additional testing is needed.
To do this, perform the following command (in a command line/powershell with admin permissions):
`mklink Path\To\Mod Organizer 2\ModOrganizer.ini %USERPROFILE%\AppData\Local\ModOrganizer\ModOrganizer.ini`

EDIT: Turns out, that's not the only compatibility issue. For example the @Binary(string) notation of current Mod Organizer is not being understood. For now I ended up copying the ModOrganizer.ini to the MO2 installation directory and fixing these issues manually.
